src/ch/epfl/leb/sass/simulator/loggers/FrameLogger.java

```diff
@@ -109,7 +110,7 @@ public void saveLogFile() throws IOException {
         FileWriter fileWriter = new FileWriter(this.filename);
         PrintWriter printWriter = new PrintWriter(fileWriter);
 
-        printWriter.println("frame,id,x,y,z,time_on");
+        printWriter.println("frame,id,x,y,z,brightness");
```

Comment:
Would you mind adding `time_on` back to the logger as a seventh column? I can imagine that we might still find this useful in the future, so having both `brightness` and `time_on` available would be nice.

Once it's added back to this file and the unit tests, I will merge the changes (assuming the updated checks pass).

Reply:
Good idea. It's done!